presto-product-tests/conf/docker/common/docker-compose.yml

```diff
@@ -0,0 +1,79 @@
+version: '2'
```

Comment:
Maybe create two common files. First for kerberos services and second for services without kerberos.